src/components/IconButton/IconButton.tsx

```diff
@@ -4,15 +4,15 @@ import styled from 'styled-components';
 
 import { IconTypes, SSCIconNames } from '../../theme/icons/icons.enums';
 import { SpacingSizeValuePropType } from '../../types/spacing.types';
-import { getFontSize, pxToRem } from '../../utils';
+import { getFontSize, getRadii, pxToRem } from '../../utils';
 import { Button } from '../Button';
 import { ButtonVariants } from '../Button/Button.enums';
 import { Icon } from '../Icon';
 import { SSCIcons, Types } from '../Icon/Icon.types';
 import { IconButtonProps } from './IconButton.types';
 
 const StyledIconButton = styled(Button)`
-  border-radius: 100%;
+  border-radius: ${getRadii('circle')};
```

Comment:
should we create a enum that holds radii values ?

Reply:
That's good idea, something similar to what we have for colors

Reply:
mmm yep.. I was thinking about it when I was doing the replacements. Let's create constants instead